src/intl/en/glossary.json

```
@@ -225,8 +233,12 @@
   "node-definition": "A software client that participates in the network. <a href=\"/developers/docs/nodes-and-clients/\">More on nodes and clients</a>.",
   "nonce-term": "Nonce",
   "nonce-definition": "In cryptography, a value that can only be used once. An account nonce is a transaction counter in each account, which is used to prevent replay attacks.",
+  "off-chain-term": "Off-Chain",
+  "off-chain-definition": "Off-chain means any transaction or data that exists outside the blockchain. Because committing every transaction on-chain can be expensive and inefficient, third-party tools like oracles that handle pricing data, or layer 2 solutions that execute a higher throughput of transactions, handle a bulk of the processing work off-chain, and will submit information on-chain at less frequent intervals.",
   "ommer-term": "Ommer (uncle) block",
   "ommer-definition": "When a proof-of-work <a href=\"/glossary/#miner\">miner</a> finds a valid <a href=\"/glossary/#block\">block</a>, another miner may have published a competing block which is added to the tip of the blockchain first. This valid, but stale, block can be included by newer blocks as <em>ommers</em> and receive a partial block reward. The term \"ommer\" is the preferred gender-neutral term for the sibling of a parent block, but this is also sometimes referred to as an \"uncle\". This was relevant for Ethereum when it was a <a href=\"/glossary/#pow\">proof-of-work</a> network, but ommers are not a feature of <a href=\"/glossary/#pos\">proof-of-stake</a> Ethereum because precisely one block proposer is selected in each slot.",
+  "on-chain-term": "On-Chain",
+  "on-chain-definition": "On-chain is an umbrella term that includes any transaction or data that is available on the blockchain and visible to all nodes on the blockchain network such as mempool data, historical transactions, and account information.",
```

cc @konopkja 
```suggestion
```

---

no. like this:

**Tooltip**
"Refers to actions or transactions that happen on the blockchain and are publicly available. "

**Full glossary definition**
"Refers to actions or transactions that happen on the blockchain and are publicly available. 

Think of it as writing something in a big, shared notebook that everyone can see and check, making sure that whatever is written (like sending digital money or making a contract) is permanent and can't be changed or erased."

---

Ok, then it would be:
```suggestion
  "on-chain-term": "On-Chain",
  "on-chain-definition": "Refers to actions or transactions that happen on the blockchain and are publicly available.<br /><br />Think of it as writing something in a big, shared notebook that everyone can see and check, making sure that whatever is written (like sending digital money or making a contract) is permanent and can't be changed or erased.",
```

The short version (for the tooltip) will be added in a different PR.

---

lgtm